plugin/src/test/java/io/jenkins/plugins/coverage/model/DockerGitForensicsITest.java

```diff
@@ -0,0 +1,294 @@
+package io.jenkins.plugins.coverage.model;
+
+import java.io.IOException;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.flow.FlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import hudson.model.FreeStyleProject;
+import hudson.model.Node;
+import hudson.model.Run;
+import hudson.plugins.git.BranchSpec;
+import hudson.plugins.git.GitSCM;
+import hudson.plugins.git.extensions.impl.RelativeTargetDirectory;
+
+import io.jenkins.plugins.coverage.CoveragePublisher;
+import io.jenkins.plugins.coverage.adapter.JacocoReportAdapter;
+import io.jenkins.plugins.util.IntegrationTestWithJenkinsPerSuite;
+
+import static io.jenkins.plugins.coverage.model.CoverageMetric.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.assertj.core.api.Assumptions.*;
+
+/**
+ * Tests the integration of the Forensics API Plugin while using its Git implementation.
+ *
+ * @author Florian Orendi
+ */
+@Testcontainers(disabledWithoutDocker = true)
+class DockerGitForensicsITest extends IntegrationTestWithJenkinsPerSuite {
```

Comment:
I think it is sufficient to use `GitForensicsITest` as there is no other implementation